app-crypt/stegsnow/stegsnow-20130616.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
```

Comment:
Shall we replace "Has an option to ..." by "It has an option to..." ?
(I'm just inserting "It" before "Has")